<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ctkHostAppExampleWidget</class>
 <widget class="QWidget" name="ctkHostAppExampleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>593</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Application Path:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="applicationPathLabel">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="runButton">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Plugin Status:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Process Status:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="processStateLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Not running</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>107</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="SuspendButton">
       <property name="text">
        <string>Suspend</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Show Messages</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>107</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QTextEdit" name="messageOutput">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="ctkHostedAppPlaceholderWidget" name="placeholderFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="crashLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>crash.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkHostedAppPlaceholderWidget</class>
   <extends>QFrame</extends>
   <header>ctkHostedAppPlaceholderWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>ctkHostAppExampleWidget</receiver>
   <slot>startButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopButton</sender>
   <signal>clicked()</signal>
   <receiver>ctkHostAppExampleWidget</receiver>
   <slot>stopButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadButton</sender>
   <signal>clicked()</signal>
   <receiver>ctkHostAppExampleWidget</receiver>
   <slot>loadButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>messageOutput</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>runButton</sender>
   <signal>clicked()</signal>
   <receiver>ctkHostAppExampleWidget</receiver>
   <slot>runButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>538</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SuspendButton</sender>
   <signal>clicked()</signal>
   <receiver>ctkHostAppExampleWidget</receiver>
   <slot>suspendButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ctkHostAppExampleWidget</receiver>
   <slot>cancelButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startButtonClicked()</slot>
  <slot>stopButtonClicked()</slot>
  <slot>loadButtonClicked()</slot>
  <slot>runButtonClicked()</slot>
  <slot>suspendButtonClicked()</slot>
  <slot>cancelButtonClicked()</slot>
 </slots>
</ui>
